SPECS/libtiff/libtiff.spec

```diff
@@ -30,13 +28,14 @@ It contains the libraries and header files to create applications
 
 %prep
 %setup -q -n libtiff-v%{version}
-%patch0 -p1
-%patch1 -p1
-%patch2 -p1
-%patch3 -p1
+
+libtoolize --force --copy
+aclocal -I ./m4
+autoheader
+automake --foreign --add-missing --copy
+autoconf
```

Comment:
NIT: is there a way for us to leverage `autosetup` here ?
maybe something on [these](https://github.com/sailfishos/libtiff/blob/master/rpm/libtiff.spec#L52) lines.

Reply:
Fixed